Formula/telnet.rb

```diff
@@ -3,6 +3,7 @@ class Telnet < Formula
   homepage "https://opensource.apple.com/"
   url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-63.tar.gz"
   sha256 "13858ef1018f41b93026302840e832c2b65289242225c5a19ce5e26f84607f15"
+  license all_of: ["BSD-4-Clause-UC", "APSL-1.0", "MIT-CMU"]
```

Comment:
```suggestion
  license all_of: ["BSD-4-Clause-UC", "APSL-1.0"]
```

Let's just get rid of the `MIT_CMU` license for now. We know that it isn't correct.

Reply:
Done!